cmd/geth/consolecmd.go

```diff
@@ -207,7 +208,7 @@ func ephemeralConsole(ctx *cli.Context) error {
 	}
 	// Wait for pending callbacks, but stop for Ctrl-C.
 	abort := make(chan os.Signal, 1)
-	signal.Notify(abort, os.Interrupt)
+	signal.Notify(abort, syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT, syscall.SIGHUP)
```

Comment:
I don't agree about SIGHUP and SIGQUIT: I use SIGQUIT all the time to get a stack dump. SIGHUP is sometimes used to tell processes to rotate logs and should not lead to process termination.

Reply:
Fair enough, I'll change the code.